generators/generator-microsoft-bot-adaptive/generators/app/index.js

```diff
@@ -8,6 +8,9 @@ const { v4: uuidv4 } = require('uuid');
 
 const options = rt.Record({
   applicationSettingsDirectory: rt.String,
+  botProjectSettings: rt.Record({
+    skills: rt.Dictionary(rt.Record, 'string').Or(rt.Undefined),
```

Comment:
What is the shape here meant to be? I would imagine `rt.Dictionary(rt.Unknown)` would work. Also, being more explicit and structured with the types here means the code is cleaner below. If the intent is that inheriting bots can specify a subset of `botProjectSettings` keys (i.e. if we add more keys in the future), then `.Or(rt.Undefined` should be omitted from the individual keys and `rt.Record({skills: rt.Dictionary(rt.Unknown)}).asPartial()` should be used, which says "make every key in this object optional".